### PR TITLE
Add dropdown divider to form dropdown

### PR DESCRIPTION
**Pull request by stephherbers**
## Product Description

Minor UI update to dropdown-- no ticket, but change suggested [here](https://dimagi-dev.atlassian.net/browse/USH-3062?focusedCommentId=261453)
Screenshot 2023-04-19 at 3 27 44 PM

## Feature Flag

divider is only shown with CHANGE_LANGUAGE_FORM

### Safety story

Just adds divider-- html + css only

### QA Plan

Not planning on QA

### Rollback instructions

- [X] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [X] Risk label is set correctly
- [X] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by Jtang-1 — APPROVED** [review]
That’s a nice touch

**Comment by stephherbers** [comment]
Update: got product feedback to scrap icon and fix spacing. I'll update the title/description to match

**Review by esoergel — APPROVED** [review]
so fast!

File: corehq/apps/cloudcare/templates/formplayer/menu_list.html [hunk 1]
```diff
@@ -131,7 +131,7 @@ <h1 class="page-title"><%- title %></h1>
   <div tabindex="0" id="form-menu-dropdown">
      <i class="fa fa-bars dropdown-toggle" data-toggle="dropdown"></i>
      <ul class="dropdown-menu dropdown-menu-right noprint-sub-container">
-       <li tabindex="0" class="dropdown-header" id="print-header">{% trans "Print" %} <i class="fa fa-print dropdown-icon"></i></li>
```
Comment (orangejenny) [inline corehq/apps/cloudcare/templates/formplayer/menu_list.html]: didn't like the icon?

Comment (orangejenny) [reply corehq/apps/cloudcare/templates/formplayer/menu_list.html]: (I liked it, just would have put it to the left like we do in the left-side menu)

Comment (stephherbers) [reply corehq/apps/cloudcare/templates/formplayer/menu_list.html]: I'm open to the icon, Cody thought it looked lonely haha-- figured it's easy enough to add in later (also yes to left side just then would need to fiddle with spacing margins a bit in the `.dropdown-header`

**Pull request merged**